### PR TITLE
ADD DALL-E

### PR DESCRIPTION
**Pull request by mishalhossin**
Add dall-e image generation uwu

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -70,4 +75,32 @@ async def imagine(interaction: discord.Interaction, prompt: str):
 except FileNotFoundError:
     print("BOT TOKEN NOT FOUND! PUT YOUR BOT TOKEN IN `bot_token.txt`")
 
+@bot.hybrid_command(name="dalleimagine", description="Generate image using DALLE")
+async def images(ctx, *, prompt):
+    url = "https://imagine.mishal0legit.repl.co"
```
Comment (mak448a) [inline bot.py:80]: What does your Replit url do? Will this stay up indefinitely?

**Review by mak448a — CHANGES_REQUESTED** [review]
Please pull in the latest main branch. Thank you!

File: bot.py (hunk 1 above)
Comment (mishalhossin) [reply bot.py:80]: yes

**Comment by mak448a** [comment]
Why did you close it?

**Pull request closed without merging**